CONTRIBUTING.md

```diff
@@ -52,79 +52,174 @@ name and contact info to the [AUTHORS](AUTHORS) file.
-4. Ensure both `DART_SDK`, `FLUTTER_SDK` and `JAVA_HOME` are added to the `PATH`
-   in the shell initialization script that runs at login.
-   (not just for the one used for every interactive shell).
-   ```shell
-   export PATH=$DART_SDK/bin:$FLUTTER_SDK/bin:$JAVA_HOME/bin:$PATH
-   ```
-5. Make sure you're using the latest stable release of IntelliJ,
-   or download and install the latest version of IntelliJ (2023.1 or later).
+      In your shell configuration file (e.g. `.bashrc` or `.zshrc`), set your `JAVA_HOME` env variable to match that version.
+      ```shell
+      export JAVA_HOME=`/usr/libexec/java_home -v 23.0.2
+      ```
+
+3. Set your `FLUTTER_SDK` path in the configuration file for your shell environment.
+    - For example, on macOS, the following works:
+      Check where your Flutter SDK is installed:
+      ```shell
+      which flutter
+      ```
+      This should print out a path to the Flutter SDK such as:
+      ```shell
+      home/path/to/flutter/bin/flutter
+      ```
+      In your shell configuration file (e.g. `.bashrc` or `.zshrc`), set your `FLUTTER_SDK` env variable to match the path.
+      ```shell
+      export FLUTTER_SDK="$HOME/path/to/flutter"
+      ```
+
+4. Set your `DARK_SDK` path in the configuration file for your shell environment.
+    - In your shell configuration file (e.g. `.bashrc` or `.zshrc`), set your `DART_SDK` env variable to match the Dart SDK in your Flutter SDK. This should look like the `FLUTTER_SDK` path (added above) with `/bin/cache/dart-sdk` appened to the end. 
+    ```shell
+    export DART_SDK="$HOME/path/to/flutter/bin/cache/dart-sdk`"
+    ```
+
+5. Add `DART_SDK`, `FLUTTER_SDK` and `JAVA_HOME` to your `PATH`.
+    - In your shell configuration file (e.g. `.bashrc` or `.zshrc`), below where your `JAVA_HOME`, `FLUTTER_SDK`, `DART_SDK` env variables were set, add the following line:
+    ```shell
+    export PATH=$DART_SDK/bin:$FLUTTER_SDK/bin:$JAVA_HOME/bin:$PATH"
+    ```
+
+6. Update your current `PATH`.
+    - Either re-start your terminal or run `source ~/.zshrc` / `source ~/.bashrc` to add the new environment variables to your `PATH`.
+
+## IntelliJ set-up
+
+1. Make sure you're using the latest stable release of IntelliJ,
+   or download and install the latest version of IntelliJ (2023.1 or later). 
     - [IntelliJ Downloads](https://www.jetbrains.com/idea/download/)
     - Either the community edition (free) or Ultimate will work.
-    - Determine the directory of your downloaded IntelliJ IDEA installation. e.g.
-        * `IntelliJ IDEA CE.app` (macOS)
-        * `~/Library/Application Support/JetBrains/Toolbox/apps/IDEA-U/ch-0/231.8109.175/IntelliJ IDEA.app` (macOS)
-        * `~/idea-IC-231.8109.175` (Linux)
-        * `X:\path\to\your\IDEA-U\ch-0\231.8109.175` (Windows after installed)
-6. Start the IntelliJ IDEA with the `flutter-intellij` project.
+
+2. [PLACHOLDER] Run [command?] to generate the `gradle.properties` file.
```

Comment:
@jwren What is the command we need to run now that `gradle.properties` is gitignored?

Reply:
Any of the ./bin/plugin commands work, many of them have a flag to specify which version.  As we discussed yesterday, ideally, we would have a new command where all it does is update and create this file

Reply:
Any of the ./bin/plugin commands work, many of them have a flag to specify which version.  As we discussed yesterday, ideally, we would have a new command where all it does is update and create this file

Reply:
Hmm I just tried testing this by deleting `gradle.properties` and running `./bin/plugin generate` but it didn't create it. I then first created `gradle.properties` and ran `./bin/plugin generate` and it didn't populate it with anything. 

Should I add instructions for manually creating a `gradle.properties` file instead? At last until we have a command to create this file

Reply:
For the time being, yes, include the gradle.properties that is currently working for you.